### ISSUE TITLE
Very high NX Daemon CPU usage

### ISSUE THREAD

**Issue by karolisKTri** [nrwl/nx#19208]
### Current Behavior

NX Daemon process uses a lot of CPU, and after it's started computer gets barely usable (and `nx graph` doesn't seem to work).
![image](https://github.com/nrwl/nx/assets/130636951/e19952d5-d25d-4a22-8438-aa288be0f5c9)

### Expected Behavior

Daemon runs normally.

### Steps to Reproduce

1. Run `nx report`.
2. Observe high CPU.

### Nx Report

```shell
>  NX  Falling back to ts-node for local typescript execution. This may be a little slower.
  - To fix this, ensure @swc-node/register and @swc/core have been installed

 >  NX   Report complete - copy this into the issue template

   Node   : 16.17.0
   OS     : win32-x64
   npm    : 8.19.3

   nx (global)        : 16.6.0
   nx                 : 16.8.1
   @nx/js             : 16.8.1
   @nx/jest           : 16.8.1
   @nx/linter         : 16.8.1
   @nx/workspace      : 16.8.1
   @nx/cypress        : 16.8.1
   @nx/devkit         : 16.8.1
   @nx/eslint-plugin  : 16.8.1
   @nrwl/tao          : 16.8.1
   @nx/web            : 16.8.1
   typescript         : 4.8.4
   ---------------------------------------
   Community plugins:
   @ngrx/effects        : 15.4.0
   @ngrx/entity         : 15.4.0
   @ngrx/store          : 15.4.0
   @ngrx/store-devtools : 15.4.0
   eslint-plugin-ngrx   : 2.1.4
```

### Failure Logs

```shell
[NX Daemon Server] - 2023-09-18T07:41:03.823Z - Started listening on: \\.\pipe\nx\C:\Users\x\AppData\Local\Temp\4d868c2730d8c0d8d33c\d.sock
[NX Daemon Server] - 2023-09-18T07:41:03.832Z - [WATCHER]: Subscribed to changes within: C:\x\x (native)
[NX Daemon Server] - 2023-09-18T07:41:03.834Z - Established a connection. Number of open connections: 1
[NX Daemon Server] - 2023-09-18T07:41:03.835Z - Established a connection. Number of open connections: 2
[NX Daemon Server] - 2023-09-18T07:41:03.839Z - [REQUEST]: Client Request for Project Graph Received
[NX Daemon Server] - 2023-09-18T07:41:10.548Z - [REQUEST]: Updated file-hasher based on watched changes, recomputing project graph...
[NX Daemon Server] - 2023-09-18T07:41:10.548Z - [REQUEST]: 
[NX Daemon Server] - 2023-09-18T07:41:10.549Z - [REQUEST]: 
[NX Daemon Server] - 2023-09-18T07:41:25.919Z - [REQUEST]: Responding to the client. project-graph
[NX Daemon Server] - 2023-09-18T07:41:25.921Z - Time taken for 'total for creating and serializing project graph' 22080.324599981308ms
[NX Daemon Server] - 2023-09-18T07:41:25.921Z - Closed a connection. Number of open connections: 1
[NX Daemon Server] - 2023-09-18T07:41:25.921Z - Done responding to the client project-graph
[NX Daemon Server] - 2023-09-18T07:41:25.921Z - Handled REQUEST_PROJECT_GRAPH. Handling time: 22080. Response time: 2.
[NX Daemon Server] - 2023-09-18T07:41:25.986Z - [WATCHER]: 2 file(s) created or restored, 0 file(s) modified, 2 file(s) deleted
'git' is not recognized as an internal or external command,
operable program or batch file.
[NX Daemon Server] - 2023-09-18T07:41:36.804Z - [REQUEST]: Updated file-hasher based on watched changes, recomputing project graph...
[NX Daemon Server] - 2023-09-18T07:41:36.804Z - [REQUEST]: 13730619769784977598,1841003217851443190
[NX Daemon Server] - 2023-09-18T07:41:36.804Z - [REQUEST]: node_modules/.cache/nx/project-graph.json~19a0fbff,node_modules/.cache/nx/file-map.json~19a0fbff
[NX Daemon Server] - 2023-09-18T07:41:44.215Z - Time taken for 'hash changed files from watcher' 535.6110000014305ms
[NX Daemon Server] - 2023-09-18T07:41:44.219Z - [REQUEST]: Client Request for Project Graph Received
[NX Daemon Server] - 2023-09-18T07:41:44.219Z - [REQUEST]: Responding to the client. project-graph
[NX Daemon Server] - 2023-09-18T07:41:44.222Z - Time taken for 'total for creating and serializing project graph' 0.26819998025894165ms
[NX Daemon Server] - 2023-09-18T07:41:44.222Z - Done responding to the client project-graph
[NX Daemon Server] - 2023-09-18T07:41:44.222Z - Handled REQUEST_PROJECT_GRAPH. Handling time: 0. Response time: 3.
[NX Daemon Server] - 2023-09-18T07:41:44.264Z - [WATCHER]: 2 file(s) created or restored, 0 file(s) modified, 2 file(s) deleted
'git' is not recognized as an internal or external command,
operable program or batch file.
[NX Daemon Server] - 2023-09-18T07:41:52.981Z - [REQUEST]: Updated file-hasher based on watched changes, recomputing project graph...
[NX Daemon Server] - 2023-09-18T07:41:52.981Z - [REQUEST]: 8181998674025597295,13730619769784977598
[NX Daemon Server] - 2023-09-18T07:41:52.982Z - [REQUEST]: node_modules/.cache/nx/project-graph.json~984988b2,node_modules/.cache/nx/file-map.json~984988b2
[NX Daemon Server] - 2023-09-18T07:42:00.865Z - Time taken for 'hash changed files from watcher' 299.3677999973297ms
[NX Daemon Server] - 2023-09-18T07:42:00.866Z - Closed a connection. Number of open connections: 0
[NX Daemon Server] - 2023-09-18T07:42:00.900Z - [WATCHER]: 2 file(s) created or restored, 0 file(s) modified, 2 file(s) deleted
'git' is not recognized as an internal or external command,
operable program or batch file.
[NX Daemon Server] - 2023-09-18T07:42:09.401Z - [REQUEST]: Updated file-hasher based on watched changes, recomputing project graph...
[NX Daemon Server] - 2023-09-18T07:42:09.401Z - [REQUEST]: 17836643716389074396,13730619769784977598
[NX Daemon Server] - 2023-09-18T07:42:09.401Z - [REQUEST]: node_modules/.cache/nx/project-graph.json~0e86991e,node_modules/.cache/nx/file-map.json~0e86991e
[NX Daemon Server] - 2023-09-18T07:42:18.449Z - Time taken for 'hash changed files from watcher' 372.02240002155304ms
[NX Daemon Server] - 2023-09-18T07:42:18.450Z - Established a connection. Number of open connections: 1
[NX Daemon Server] - 2023-09-18T07:42:18.451Z - Closed a connection. Number of open connections: 0
[NX Daemon Server] - 2023-09-18T07:42:18.495Z - [WATCHER]: 2 file(s) created or restored, 0 file(s) modified, 2 file(s) deleted
'git' is not recognized as an internal or external command,
operable program or batch file.
[NX Daemon Server] - 2023-09-18T07:42:27.574Z - [REQUEST]: Updated file-hasher based on watched changes, recomputing project graph...
[NX Daemon Server] - 2023-09-18T07:42:27.574Z - [REQUEST]: 8034090811421896697,13730619769784977598
[NX Daemon Server] - 2023-09-18T07:42:27.574Z - [REQUEST]: node_modules/.cache/nx/file-map.json~aa2ce479,node_modules/.cache/nx/project-graph.json~aa2ce479
[NX Daemon Server] - 2023-09-18T07:42:35.462Z - Time taken for 'hash changed files from watcher' 303.06640005111694ms
[NX Daemon Server] - 2023-09-18T07:42:35.500Z - [WATCHER]: 2 file(s) created or restored, 0 file(s) modified, 2 file(s) deleted
'git' is not recognized as an internal or external command,
operable program or batch file.
[NX Daemon Server] - 2023-09-18T07:42:43.225Z - [REQUEST]: Updated file-hasher based on watched changes, recomputing project graph...
[NX Daemon Server] - 2023-09-18T07:42:43.226Z - [REQUEST]: 13730619769784977598,18257046504176345281
[NX Daemon Server] - 2023-09-18T07:42:43.226Z - [REQUEST]: node_modules/.cache/nx/file-map.json~67a36186,node_modules/.cache/nx/project-graph.json~67a36186
[NX Daemon Server] - 2023-09-18T07:42:51.669Z - Time taken for 'hash changed files from watcher' 295.9414999485016ms
[NX Daemon Server] - 2023-09-18T07:42:51.707Z - [WATCHER]: 2 file(s) created or restored, 0 file(s) modified, 2 file(s) deleted
'git' is not recognized as an internal or external command,
operable program or batch file.
[NX Daemon Server] - 2023-09-18T07:43:01.116Z - [REQUEST]: Updated file-hasher based on watched changes, recomputing project graph...
[NX Daemon Server] - 2023-09-18T07:43:01.117Z - [REQUEST]: 17964717694076000995,13730619769784977598
[NX Daemon Server] - 2023-09-18T07:43:01.117Z - [REQUEST]: node_modules/.cache/nx/file-map.json~6b12774d,node_modules/.cache/nx/project-graph.json~6b12774d
[...]
[NX Daemon Server] - 2023-09-18T08:26:43.548Z - [WATCHER]: Processing file changes in outputs
[NX Daemon Server] - 2023-09-18T08:26:43.548Z - [WATCHER]: Stopping the watcher for C:\xcity\Work\LocusCloud\FrontEnd (outputs)
[NX Daemon Server] - 2023-09-18T08:26:43.548Z - Server stopped because: "this process is no longer the current daemon (native)"
[NX Daemon Server] - 2023-09-18T08:26:51.722Z - [REQUEST]: Responding to the client. project-graph
[NX Daemon Server] - 2023-09-18T08:26:51.724Z - Time taken for 'total for creating and serializing project graph' 35912.92040002346ms
[NX Daemon Server] - 2023-09-18T08:26:51.732Z - [WATCHER]: nx.json was modified
[NX Daemon Server] - 2023-09-18T08:26:51.735Z - [WATCHER]: 0 file(s) created or restored, 0 file(s) modified, 2 file(s) deleted
[NX Daemon Server] - 2023-09-18T08:26:51.739Z - [WATCHER]: .idea/.idea.FrontEnd.dir/.idea/workspace.xml was created or restored
[NX Daemon Server] - 2023-09-18T08:26:51.742Z - [WATCHER]: 2 file(s) created or restored, 1 file(s) modified, 2 file(s) deleted
[NX Daemon Server] - 2023-09-18T08:26:51.745Z - [WATCHER]: node_modules/.cache/nx/d/daemon.log was modified
[NX Daemon Server] - 2023-09-18T08:26:51.752Z - [WATCHER]: 2 file(s) created or restored, 1 file(s) modified, 2 file(s) deleted
[NX Daemon Server] - 2023-09-18T08:26:51.752Z - [WATCHER]: Processing file changes in outputs
[NX Daemon Server] - 2023-09-18T08:26:51.753Z - [WATCHER]: Processing file changes in outputs
[NX Daemon Server] - 2023-09-18T08:26:51.753Z - [WATCHER]: Processing file changes in outputs
[NX Daemon Server] - 2023-09-18T08:26:51.754Z - Closed a connection. Number of open connections: 1
[NX Daemon Server] - 2023-09-18T08:26:51.755Z - Done responding to the client project-graph
[NX Daemon Server] - 2023-09-18T08:26:51.755Z - Handled REQUEST_PROJECT_GRAPH. Handling time: 35914. Response time: 33.
[NX Daemon Server] - 2023-09-18T08:26:51.759Z - Established a connection. Number of open connections: 2
[NX Daemon Server] - 2023-09-18T08:26:51.760Z - Established a connection. Number of open connections: 3
[NX Daemon Server] - 2023-09-18T08:26:51.764Z - [REQUEST]: Client Request for Project Graph Received
'git' is not recognized as an internal or external command,
operable program or batch file.
[NX Daemon Server] - 2023-09-18T08:26:52.145Z - [REQUEST]: Responding to the client. handleRequestFileData
[NX Daemon Server] - 2023-09-18T08:27:02.900Z - [REQUEST]: Updated file-hasher based on watched changes, recomputing project graph...
[NX Daemon Server] - 2023-09-18T08:27:02.900Z - [REQUEST]: 8659585175887656111,16634400829989264251,8629832571267616389,3031922968256640715,16384808305471860760
[NX Daemon Server] - 2023-09-18T08:27:02.901Z - [REQUEST]: .idea/.idea.FrontEnd.dir/.idea/workspace.xml.tmp,node_modules/.cache/nx/project-graph.json~76dadda1,node_modules/.cache/nx/file-map.json~76dadda1,node_modules/.cache/nx/project-graph.json~ac66e879,node_modules/.cache/nx/file-map.json~ac66e879
```

### Operating System

- [ ] macOS
- [ ] Linux
- [X] Windows
- [ ] Other (Please specify)

### Additional Information

We are migrating our Angular apps and libraries to NX monorepo and faced this issue. For now we migrated one of two apps and three libraries. ~~Maybe it could be related to the fact that we don't use GIT (instead we use TFS) and it's not installed on my machine.~~ EDIT: I tried this https://github.com/nrwl/nx-examples and it works fine. Probably something is wrong with my configuration.
~~Probably related - I tried `nx graph` command and it seems not to work correctly. I get flat repo structure - without any relations.~~ EDIT: I partially fixed it - problem is that that I had custom "baseUrl" value in `tsconfig.json` files and probably `nx graph` can't handle that.
![image](https://github.com/nrwl/nx/assets/130636951/99f5dd79-19d4-4158-9f10-2fd52bf63d18)

 When I tried to run this command with `useDaemonProcess: true` I got an error, here's `daemon-error.log`:
```
Error: write EPIPE

Messages from the log:
[NX Daemon Server] - 2023-09-18T08:00:14.559Z - [WATCHER]: node_modules/.cache/nx/d/server-process.json was modified
[NX Daemon Server] - 2023-09-18T08:00:14.560Z - Closed a connection. Number of open connections: 1
[NX Daemon Server] - 2023-09-18T08:00:14.560Z - Done responding to the client project-graph
[NX Daemon Server] - 2023-09-18T08:00:14.560Z - Handled REQUEST_PROJECT_GRAPH. Handling time: 34461. Response time: 15.
[NX Daemon Server] - 2023-09-18T08:00:14.561Z - [WATCHER]: Stopping the watcher for C:\x\x\ (sources)
[NX Daemon Server] - 2023-09-18T08:00:14.561Z - [WATCHER]: Stopping the watcher for C:\x\x\ (outputs)
[NX Daemon Server] - 2023-09-18T08:00:14.561Z - Server stopped because: "this process is no longer the current daemon (native)"
[NX Daemon Server] - 2023-09-18T08:00:15.858Z - Started listening on: \\.\pipe\nx\C:\Users\x\AppData\Local\Temp\1556985791ceecfde5d3\d.sock
[NX Daemon Server] - 2023-09-18T08:00:15.863Z - [WATCHER]: Subscribed to changes within: C:/x\x\ (native)
[NX Daemon Server] - 2023-09-18T08:00:15.866Z - Established a connection. Number of open connections: 1
[NX Daemon Server] - 2023-09-18T08:00:15.867Z - Established a connection. Number of open connections: 2
[NX Daemon Server] - 2023-09-18T08:00:16.959Z - [REQUEST]: Responding to the client. handleRequestFileData
[NX Daemon Server] - 2023-09-18T08:00:16.963Z - Closed a connection. Number of open connections: 1
[NX Daemon Server] - 2023-09-18T08:00:16.964Z - Done responding to the client handleRequestFileData
[NX Daemon Server] - 2023-09-18T08:00:16.965Z - Handled REQUEST_FILE_DATA. Handling time: 1086. Response time: 6.
[NX Daemon Server] - 2023-09-18T08:00:21.547Z - [WATCHER]: node_modules/.cache/nx/d/daemon.log was modified
[NX Daemon Server] - 2023-09-18T08:00:21.603Z - [WATCHER]: node_modules/.cache/nx/d/daemon.log was modified
'git' is not recognized as an internal or external command,
operable program or batch file.



More information: C:\x\x\\node_modules\.cache\nx\d\daemon.log
```
Then occasionally get this error (in this case I ran `nx serve` and then `nx graph`):
```
 >  NX   Daemon process terminated and closed the connection

   Please rerun the command, which will restart the daemon.
   If you get this error again, check for any errors in the daemon process logs found in: C:\x\x\node_modules\.cache\nx\d\daemon.log
```
Also I suspect there might be something wrong with configuration as I was doing migration manually. Here is `nx.json`:
```
{
    "$schema": "./node_modules/nx/schemas/nx-schema.json",
    "tasksRunnerOptions": {
        "default": {
            "runner": "nx/tasks-runners/default",
            "options": {
                "cacheableOperations": ["build", "lint", "test"],
                "useDaemonProcess": true
            }
        }
    },
    "targetDefaults": {
        "build": {
            "dependsOn": ["^build"],
            "inputs": ["production", "^production"]
        },
        "test": {
            "inputs": ["default", "^production"]
        },
        "lint": {
            "inputs": ["default", "{workspaceRoot}/.eslintrc.json", "{workspaceRoot}/.eslintignore"]
        }
    },
    "namedInputs": {
        "default": ["{projectRoot}/**/*", "sharedGlobals"],
        "production": [
            "default",
            "!{projectRoot}/**/?(*.)+(spec|test).[jt]s?(x)?(.snap)",
            "!{projectRoot}/tsconfig.spec.json",
            "!{projectRoot}/src/test-setup.[jt]s",
            "!{projectRoot}/test-setup.[jt]s",
            "!{projectRoot}/.eslintrc.json"
        ],
        "sharedGlobals": []
    },
    "generators": {
        "@nx/angular:application": {
            "style": "scss",
            "linter": "eslint",
            "unitTestRunner": "karma"
        },
        "@nx/angular:library": {
            "linter": "eslint",
            "unitTestRunner": "karma"
        },
        "@nx/angular:component": {
            "style": "scss"
        }
    }
}
```
`tsconfig.base.json`:
```
{
    "compileOnSave": false,
    "angularCompilerOptions": {
        "strictTemplates": true,
        "strictInputTypes": true,
        "strictMetadataEmit": true,
        "enableResourceInlining": true,
        "fullTemplateTypeCheck": true,
        "strict": true,
        "skipTemplateCodegen": true,
        "strictInjectionParameters": true,
        "strictInputAccessModifiers": true
    },
    "compilerOptions": {
        "esModuleInterop": true,
        "allowSyntheticDefaultImports": true,
        "outDir": "./dist/out-tsc",
        "sourceMap": true,
        "declaration": false,
        "downlevelIteration": true,
        "experimentalDecorators": true,
        "module": "esnext",
        "moduleResolution": "node",
        "importHelpers": true,
        "typeRoots": ["node_modules/@types"],
        "lib": ["es2019", "dom"],
        "strict": true,
        "plugins": [
            {
                "name": "typescript-tslint-plugin"
            }
        ],
        "paths": {
            "@x/lib1": ["../../libs/lib1/src/index.ts"],
            "@x/lib1/*": ["../../libs/lib1/src/*"],
            "@x/lib2": ["../../libs/lib2/src/index.ts"],
            "@x/lib2/*": ["../../libs/lib2/src/*"],
            "@x/lib3": ["../../libs/lib3/src/index.ts"],
            "@x/lib3/*": ["../../libs/lib3/src/*"]
        },
        "useDefineForClassFields": false,
        "rootDir": ".",
        "emitDecoratorMetadata": true,
        "target": "es2015",
        "skipLibCheck": true,
        "skipDefaultLibCheck": true,
        "baseUrl": "./"
    },
    "exclude": ["node_modules", "tmp"]
}
```
I can't share sources, but let me know if I can share any other useful configuration or I could try something to get around this issue.

To summarize, I convinced my colleagues to move to monorepo because we would have faster development cycles. But for now it slower as, for example, `serve` command takes much longer. Previously it tooks 45 seconds. Now it takes twice as long. And I'm about to migrate the much bigger app to this monorepo.

**Comment by karolisKTri** [comment]
Also now I'm running unit tests in terminal and my computer almost freezes:
![image](https://github.com/nrwl/nx/assets/130636951/670ac7e4-b8ac-4a5b-bc21-3d2905b9fa2d)

**Comment by aikrez** [comment]
Similar issue when just opening IntelliJ and not opening any file
![image](https://github.com/nrwl/nx/assets/8655340/5fcf0c0a-a38f-4123-aed2-1f0f2ed41b70)

 ```
Node   : 18.16.0
   OS     : win32-x64
   yarn   : 1.22.19

   nx                 : 16.6.0
   @nx/js             : 16.6.0
   @nx/jest           : 16.6.0
   @nx/linter         : 16.6.0
   @nx/workspace      : 16.6.0
   @nx/angular        : 16.6.0
   @nx/cypress        : 16.6.0
   @nx/devkit         : 16.6.0
   @nx/eslint-plugin  : 16.6.0
   @nx/node           : 16.6.0
   @nx/plugin         : 16.6.0
   @nx/storybook      : 16.6.0
   @nrwl/tao          : 16.6.0
   @nx/web            : 16.6.0
   @nx/webpack        : 16.6.0
   typescript         : 5.1.6
```

Disabling the daemon  with  "useDaemonProcess": false stops the excessive node usage

**Comment by karolisKTri** [comment]
> Disabling the daemon with "useDaemonProcess": false stops the excessive node usage

Just a theory, but I think that NX Console plugin could start the daemon.

**Comment by aikrez** [comment]
> > Disabling the daemon with "useDaemonProcess": false stops the excessive node usage
> 
> Just a theory, but I think that NX Console plugin could start the daemon. Same happens to me.

You are correct. I disabled the plugin and no issues with the node process anymore.
This explains why I just recently had this problem after removed the 3rd party plugin and started using the Nx plugin.

**Comment by karolisKTri** [comment]
> > > Disabling the daemon with "useDaemonProcess": false stops the excessive node usage
> > 
> > 
> > Just a theory, but I think that NX Console plugin could start the daemon. Same happens to me.
> 
> You are correct. I disabled the plugin and no issues with the node process anymore. This explains why I just recently had this problem after removed the 3rd party plugin and started using the Nx plugin.

Is `nx graph` working fine for you?

**Comment by aikrez** [comment]
> Is `nx graph` working fine for you?

Yes it does

**Comment by karolisKTri** [comment]
I found the cause of this. As we don't use GIT I deleted .gitignore file which apparently excludes not only files for git but is also used by NX to determine which file changes should be tracked. This way it falls to infinite loop while reading it's own log file (which is in node_modules). Instead of .gitignore I added .nxignore, which also does the job but is more explicit. So I guess one could say that this behavior is "by design" and it's users fault so this issue can be closed or it's type could be changed. But I'll leave that for others to decide.

**Comment by AgentEnder** [comment]
@Cammisuli can you look into this?

**Comment by MillerSvt** [comment]
Addendum .nxignore didn't help me. Perhaps there are more reasons.

.nxignore
```
node_modules
```

**Comment by karolisKTri** [comment]
> Addendum .nxignore didn't help me. Perhaps there are more reasons.
> 
> .nxignore
> 
> ```
> node_modules
> ```
Try to put ```/node_modules
``` to .nxignore.

**Comment by karolisKTri** [comment]
Try to add '/' before 'node_modules', like I wrote.

**Comment by MillerSvt** [comment]
I tried.

> Still didn't help.

Sorry, I made mistake while copying nxignore.

Here's correct:

```
/node_modules
```

Also I tried to remove nxignore and change `node_modules` to `/node_modules` in gitignore.

**Comment by karolisKTri** [comment]
From looking at logs it seems that it's the same problem, so probably there's still something wrong with ignoring node_modules. I personally fixed problem when comparing how this project is implemented: https://github.com/nrwl/nx-examples.
I would check if files are really in correct location and try again to copy paste this file: https://github.com/nrwl/nx-examples/blob/master/.gitignore contents.

**Comment by MillerSvt** [comment]
I got it. There is a stable case to reproduce infinity loop:

Run `node_modules\nx\src\daemon\server\start.js` with environment:
```
NX_CACHE_PROJECTS_CONFIG=false;
NX_NATIVE_WATCHER=true;
NX_PROJECT_GLOB_CACHE=false;
NX_WORKSPACE_ROOT_PATH=C:/Users/sviat/IdeaProjects/yaklass/frontend
```

Run client script:
```
const client = require('nx/src/daemon/client/client.js');

client.daemonClient.getProjectGraph().then(() => console.log(123));
```

The server goes into an infinity loop. 
<img width="697" alt="image" src="https://github.com/nrwl/nx/assets/143048525/42ed38df-b3e5-4879-814a-b968c1504804">

The problem is reproduced when sending a request REQUEST_PROJECT_GRAPH to server and when legacy watcher is enabled. With regular watcher all is ok. I think it is not a problem with ignorefile.

I continue to investigate the problem...

**Comment by Cammisuli** [comment]
I've noticed this quite recently too, and came to the same conclusions. There's something weird happening with Nx Console for Intellij + the daemon. I think we need to fix something in the source of console. I'll check out whats new in the linked PR too.
